src/Simplex/Chat/Messages.hs

```diff
@@ -495,6 +495,16 @@ ciDeleteModeToText = \case
   CIDMBroadcast -> "this item is deleted (broadcast)"
   CIDMInternal -> "this item is deleted (internal)"
 
+ciGroupInvitationToText :: CIGroupInfo -> GroupMemberRole -> Text
+ciGroupInvitationToText CIGroupInfo {groupProfile = GroupProfile {displayName, fullName}} memberRole =
+  "invitation to join group " <> displayName <> optionalFullName displayName fullName <> " as " <> memberRoleToText memberRole
+  where
+    memberRoleToText :: GroupMemberRole -> Text
+    memberRoleToText = \case
+      GRMember -> "member"
+      GRAdmin -> "admin"
+      GROwner -> "owner"
```

Comment:
+ There is the same function in View.hs. I don't see the reason not to use encoding, tbh, for both - this text will not show in mobile UI - it's here as a fallback and to show in chat previews (which is actually bad - we should be localising those instead of using default texts).